### ISSUE TITLE
Preview of text files in the right column

### ISSUE THREAD

**Issue by lenormf** [jarun/nnn#126]
Hi,

Right now file names are being displayed in a single column, which can create some dead space in a large terminal.

Could a preview (like `head` or something) of the file currently under the cursor be displayed in that dead space, when there's any? It's convenient to manually locate a file in a directory without opening all of them.

Since it would create a lot of additional computation, maybe this feature could be opt-in?

Thanks.

**Comment by lenormf** [comment]
I'll post the patch here for anybody who would like that feature as well, when I have one.

**Comment by Samulus** [comment]
I agree with lenormf. A few lines of code to display file previews is trivial I don't see how it's possible to have """insufficient bandwidth"" for a patch like this.

**Comment by jarun** [comment]
@lenormf

> I'll post the patch here

I don't see any issues with that. Just that it won't be officially supported is all.

**Comment by jarun** [comment]
@lenormf @Samulus previews are available in master.
